metadata.yaml

```
@@ -27,7 +27,7 @@ resources:
   alertmanager-image:
     type: oci-image
     description: OCI image for alertmanager
-    upstream-source: ubuntu/prometheus-alertmanager
+    upstream-source: ubuntu/prometheus-alertmanager@sha256:30489547ce93a38e3cb3c76d102886e12a72145727f08ff690d2da0db697313d
```

`sha256:304..13d` is for `amd64`:
https://hub.docker.com/layers/ubuntu/prometheus-alertmanager/0.21-20.04_beta/images/sha256-30489547ce93a38e3cb3c76d102886e12a72145727f08ff690d2da0db697313d?context=explore

Wouldn't it make more sense to use a tag here instead? Sure, a tag is not as trustworthy as a hash, but an arch-specific hash is probably the wrong way to go.

The docker hub url itself shows this clearly:
`hub.docker.com/.../0.21-20.04_beta/images/sha256-304...13d`
```suggestion
    upstream-source: ubuntu/prometheus-alertmanager:0.21-20.04_beta
```

WDYT? @simskij @mmanciop